connections/sim/sim_connection_factory.py

```diff
@@ -21,17 +22,37 @@ def requiresBaudRate(self) -> bool:
         return False
 
     def construct(
-        self,
-        comPort: Union[int, str, None] = None,
-        baudRate: Union[int, None] = None,
-        rocket=None,
+            self,
+            comPort: Union[int, str, None] = None,
+            baudRate: Union[int, None] = None,
+            rocket=None,
     ):
         assert rocket is not None
-        executableName = EXECUTABLE_NAME
+
+        # Check FW (child) dir and FLARE (neighbour) dir for rocket build files
+        # If multiple build files found throw exception
+        flarePath = os.path.join(Path(LOCAL).parent, 'FLARE', 'avionics', 'build')
+        localPath = os.path.join(os.curdir, 'FW')
```

Comment:
```
Traceback (most recent call last):
  File "B:\Documents\CODE\UBCRocketGroundStation\com_window\main.py", line 68, in doneButtonPressed
    rocket=rocket,
  File "B:\Documents\CODE\UBCRocketGroundStation\connections\sim\sim_connection_factory.py", line 40, in construct
    childBuildFileExists = "program" in os.listdir(localPath)
FileNotFoundError: [WinError 3] The system cannot find the path specified: '.\\FW'
```
Can we please use `LOCAL` for this one too. `os.curdir` is just a relative path and wont give the correct result if the program is started from a different directory. Might also have issues with PyInstaller.

Reply:
```suggestion
        localPath = os.path.join(LOCAL, 'FW')
```